src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = props => (
+  <>
+    <div className="movies">
+      {Object.keys(props)
+        .map(movie => (
+          <MovieCard
+            key={props[movie].imdbId}
+            {...props[movie]}
```

Comment:
the same, it is better to pass named props

Reply:
Agree with everything except here,  can I just pass it like {...movie} this would pass all the properties and in case I would need to add one at MovieCard  I would not need to come back here to write an additional prop.

Reply:
if you don't use everything from the object, it's better to use those values you really need
in this case you need to have props separately